ddtrace/contrib/dbapi/__init__.py

```diff
@@ -149,7 +149,11 @@ def callproc(self, proc, *args):
         return self._trace_method(self.__wrapped__.callproc, self._self_datadog_name, proc, {}, None, proc, *args)
 
     def _set_post_execute_tags(self, span):
-        row_count = self.__wrapped__.rowcount
+        # rowcount is in the dbapi specification (https://peps.python.org/pep-0249/#rowcount)
+        # but some database drivers (cassandra-driver specifically) don't implement it.
+        row_count = getattr(self.__wrapped__, "rowcount", None)
+        if row_count is None:
```

Comment:
Can we do the same if `rowcount` is -1 which is in the spec?

```
The attribute is -1 in case no [.execute*()](https://peps.python.org/pep-0249/#id19) has been performed on the cursor or the rowcount of the last operation is cannot be determined by the interface.
```